src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+function calculateMenAverageAge(people, century = true) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person => person.sex === 'm');
+  const menInCentury = men
+    .filter((person) =>
+      Math.ceil(person.died / 100) === century || century === true);
+
+  const deathYears = menInCentury.reduce((a, b) => (a + b.died), 0);
+  const bornYears = menInCentury.reduce((a, b) => (a + b.born), 0);
```

Comment:
you can do it with one iteration instead of 2
just calculate died - born inside the reduce callback and get the sum of this value